router_v1/GluexRouter.sol

```diff
@@ -62,13 +62,12 @@ contract GluexRouter is EthReceiver {
         address payable outputReceiver; // Address to receive the output token
         address payable partnerAddress; // Address of the partner receiving surplus share
         uint256 inputAmount; // Amount of input token
-        uint256 outputAmount; // Original output amount
-        uint256 partnerFee; // Fee for the partner
-        uint256 routingFee; // Fee for the routing operation
-        uint256 externalOutputAmount; // External output amount for comparison
-        uint256 partnerSurplus; // Percentage (in bps) of surplus shared with the partner
-        uint256 routingSurplus; // Percentage (in bps) of surplus shared with the treasury
-        uint256 effectiveOutputAmount; // Effective output after deducting the routing fee
+        uint256 outputAmount; // Optimizer output amount
+        uint256 partnerFee; // Fee charged by the partner
+        uint256 routingFee; // Fee charged for routing operation
+        uint256 partnerSurplusShare; // Percentage (in bps) of surplus shared with the partner
+        uint256 routingSurplusShare; // Percentage (in bps) of surplus shared with GlueX
```

Comment:
let's call this protocolSurplusShare to prevent people misinterpret this as an upfront routing fee